cli/tenzir_platform/helpers/exceptions.py

```diff
@@ -0,0 +1,8 @@
+class PlatformCliError(Exception):
+    def __init__(self, error: str):
+        super().__init__(error)
+        self.error = error
+        self.hints: list[str] = []
+
+    def add_hint(self, hint: str):
+        self.hints.append(hint)
```

Comment:
Would you mind changing your editor config to keep/add the newline after the last line? I believe everybody else on the team has that.

Reply:
In theory `black` should be responsible for inserting and enforcing these automatically, but it seems that the check isn't configured for this repository yet.